src/cool_seq_tool/sources/uta_database.py

```diff
@@ -378,6 +378,21 @@ async def validate_genomic_ac(self, ac: str) -> bool:
         result = await self.execute_query(query)
         return result[0][0]
 
+    async def validate_mane_transcript_acc(self, transcript_list: list[dict]) -> bool:
```

Comment:
To be consistent with naming in CST, I'd recommend just using `ac` when referring to `accession`
```suggestion
    async def validate_mane_transcript_ac(self, transcript_list: list[dict]) -> bool:
```